From e02f23eef784445c585666257dd8fc181c9f45ca Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Mon, 27 Oct 2014 17:15:42 +0100
Subject: [PATCH] units: tmpfiles-setup-dev - allow unsafe file creation to
 happen in /dev at boot

This will allow us to mark static device nodes with '!' to indicate that they should only be created at early boot.

(cherry picked from commit 8c94052ee543c3598a3c7b0c46688150aa2c6168)
---
 units/systemd-tmpfiles-setup-dev.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-tmpfiles-setup-dev.service.in b/units/systemd-tmpfiles-setup-dev.service.in
index 11c5ce24ea..4ba33fd39f 100644
--- a/units/systemd-tmpfiles-setup-dev.service.in
+++ b/units/systemd-tmpfiles-setup-dev.service.in
@@ -15,4 +15,4 @@ ConditionCapability=CAP_SYS_MODULE
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=@rootbindir@/systemd-tmpfiles --prefix=/dev --create
+ExecStart=@rootbindir@/systemd-tmpfiles --prefix=/dev --create --boot
